packages/runner-ct/src/app/app.scss

```diff
@@ -135,59 +135,36 @@ main.app-ct {
 }
 
 
-// styles for react-split-pane
+// styles for react-split-pane and custom ResizableBox
 .Resizer {
-  $resize-border-color: rgba(255, 255, 255, 0);
-  $resize-hover-border-color: lighten($resize-border-color, 0.5);
-  $resize-border: 5px solid $resize-border-color;
-  $resize-hover-border: 5px solid $resize-hover-border-color;
-  $resize-thickness: 11px;
-
-  background: $resize-border-color;
-  opacity: 0.2;
+  $resize-thickness: 12px;
+  border-left: 5px solid transparent;
+  border-right: 5px solid transparent;
+  margin: -5px;
+
+  background: transparent;
   z-index: 1;
   box-sizing: border-box;
   background-clip: padding-box;
 
-  :hover {
-    transition: all 2s ease;
+  &:active {
+    background-color: lighten(#4299e1, 10%);
+    transition: background-color .3s ease-in-out;
   }
```

Comment:
I believe this should become blue on hover to signify interactivity. Instead of on drag. @CypressAdam ?

Reply:
Ya. Agree. Maybe a slightly lighter blue on hover and then as it is on grab/drag?